lib/core/utils.js

```diff
@@ -361,11 +361,20 @@ RESUtils.thingsContainer = function(body) {
 
 	return container;
 };
-RESUtils.things = function(container) {
+RESUtils.$things = function(container) {
 	container = container || RESUtils.thingsContainer();
 	return $(container).find(RESUtils.thing.prototype.thingSelector);
 };
+RESUtils.things = function(container) {
```

Comment:
I'm a bit confused... what's the difference between these two functions? Is there any reason why we'd call the first?